students/sunhobaik/users/views.py

```diff
@@ -7,33 +7,27 @@
 class UserView(View):
     def post(self, request):
         data=json.loads(request.body)
-        # p = re.compile("^[a-zA-Z0-9+-_.]+@[a-zA-z0-9-]+\.[a-zA-z0-9-]+$")
-        # # q = re.compile("^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$")
-        # if p.match(data['email']) == None :
-        #     return JsonResponse({'message':'check your email address'}, status=404)   
-        # else:
-        #     User.objects.create(
-        #         name          = data['name'],
-        #         email         = data['email'],
-        #         password      = data['password'],
-        #         phone_numbers = data['phone_numbers'],
-        #         web_site      = data['website'],
-        #         nick_name     = data['nick_name'],
-        #         address       = data['address'],
-        #         introduce     = data['introduce']
-        #     )
-        #     return JsonResponse({'message': 'created'}, status=201)
-        # elif data['email'] =='' or data['password']=='' or  data['address']=='' or  data['name']=='':
-        #     return JsonResponse({'message':'check your profile'}, status=404)
+        Email = re.compile("^[a-zA-Z0-9+-_.]+@[a-zA-z0-9-]+\.[a-zA-z0-9-]+$")
+        PW    = re.compile("^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$")
+        
+        if  PW.match(data['password'])  == None and Email.match(data['email']) == None:
+            return JsonResponse({'message':'Please check your password and email address!'}, status=400)         
```

Comment:
```suggestion
        if  PW.match(data['password']) is None or Email.match(data['email']) is None:
            return JsonResponse({'message':'Please check your password and email address!'}, status=400)         
```
None값 비교는 Is로 하는 게 일반적입니다.
and 조건보다 or 조건을 사용해주시는 게 더 효율적이겠죠?
친절한 에러메세지 좋습니다! 하지만 해당 메세지는 유저가 아닌, 프론트엔드 개발자가 보는 메세지입니다. INVALID_FORMAT 정도로만 알려주셔도 충분합니다.